cli/src/main/kotlin/com/bazel_diff/cli/GenerateHashesCommand.kt

```diff
@@ -136,6 +136,14 @@ class GenerateHashesCommand : Callable<Int> {
     )
     var ignoredRuleHashingAttributes: Set<String> = emptySet()
 
+    @CommandLine.Option(
+        names = ["-d", "--depsFile"],
```

Comment:
I wonder if we should have a longer flag name here that is more descriptive?

Reply:
I updated it to `--depEdgesFile`. Is that sufficient / do you have any other suggestions?

Reply:
That sounds good